docs/docs-ref-autogen/excel/excelscript/excelscript.table.yml

```diff
@@ -163,9 +163,7 @@ methods:
     uid: 'ExcelScript!ExcelScript.Table#getColumn:member(1)'
     package: ExcelScript!
     fullName: getColumn(key)
-    summary: >-
-      Gets a column object by name or ID. If the column doesn't exist, then this function will return an object with its
-      `isNullObject` property set to `true`<!-- -->.
+    summary: 'Gets a column object by name or ID. If the column doesn''t exist, then this function returns `undefined`<!-- -->.'
```

Comment:
```suggestion
    summary: 'Gets a column object by name or ID. If the column doesn't exist, then this function returns `undefined`<!-- -->.'
```

Reply:
These are escape characters generated by the tooling for one-line description support. They don't render in the resulting markdown: https://review.docs.microsoft.com/en-us/javascript/api/office-scripts/excelscript/excelscript.table?branch=AlexJ-Intellisense11-11&view=office-scripts#getColumn_key_